draft-ietf-core-dynlink.md

```diff
@@ -106,6 +105,7 @@ Conditional Control Attributes are defined below:
 | Attribute         | Parameter | Value            |
 | Minimum Period (s)| pmin      | xs:decimal (>0) |
 | Maximum Period (s)| pmax      | xs:decimal (>0) |
+| Change Step       | st        | xs:decimal (>0) |
```

Comment:
I added change step to this table to keep it consistent with its usage together with pmin and pmax. Does this sound good?

Reply:
Should 'st' be in the Notification Attributes table?